deploy/packages/serial-libraries.yaml

```
@@ -54,7 +54,8 @@ packages:
     specs:
       - assimp ^boost@1.65.0
       - boost
-      - deflect ^boost@1.65.0
+      - brion  ^boost@1.65.0
```

```suggestion
      - brion
```
Since you added the explicit dependency in the package.